service/resources/helpers/helpers.py

```diff
@@ -80,57 +74,57 @@ def get_custom_args(custom_args):
-            mail_settings.bypass_list_management = BypassListManagement(
-                settings['bypass_list_management']['enable'])
-        if 'footer' in settings.keys() and settings['footer'] is not None:
-            mail_settings.footer_settings = FooterSettings(
-                settings['footer']['enable'],
-                FooterText(settings['footer']['text']),
-                FooterHtml(settings['footer']['html'])
-            )
-        if 'sandbox_mode' in settings.keys() and settings['sandbox_mode'] is not None:
-            mail_settings.sandbox_mode = SandBoxMode(settings['sandbox_mode']['enable'])
-        if 'spam_check' in settings.keys() and settings['spam_check'] is not None:
-            mail_settings.spam_check = SpamCheck(
-                settings['spam_check']['enable'],
-                SpamThreshold(settings['spam_check']['threshold']),
-                SpamUrl(settings['spam_check']['post_to_url'])
-            )
-        return mail_settings
+    # @staticmethod
+    # def get_mail_settings(settings):
+    #     """ This function sets up email settings """
+    #     mail_settings = MailSettings()
+    #     if 'bcc' in settings.keys() and settings['bcc'] is not None:
+    #         mail_settings.bcc_settings = BccSettings(
+    #             settings['bcc']['enable'],
+    #             BccSettingsEmail(settings['bcc']['email'])
+    #         )
+    #     if 'bypass_list_management' in settings.keys() and settings['bypass_list_management'] is not None:
+    #         mail_settings.bypass_list_management = BypassListManagement(
+    #             settings['bypass_list_management']['enable'])
+    #     if 'footer' in settings.keys() and settings['footer'] is not None:
+    #         mail_settings.footer_settings = FooterSettings(
+    #             settings['footer']['enable'],
+    #             FooterText(settings['footer']['text']),
+    #             FooterHtml(settings['footer']['html'])
+    #         )
+    #     if 'sandbox_mode' in settings.keys() and settings['sandbox_mode'] is not None:
+    #         mail_settings.sandbox_mode = SandBoxMode(settings['sandbox_mode']['enable'])
+    #     if 'spam_check' in settings.keys() and settings['spam_check'] is not None:
+    #         mail_settings.spam_check = SpamCheck(
+    #             settings['spam_check']['enable'],
+    #             SpamThreshold(settings['spam_check']['threshold']),
+    #             SpamUrl(settings['spam_check']['post_to_url'])
+    #         )
+    #     return mail_settings
 
-    @staticmethod
-    def get_sections(sections):
-        """ This function sets up email Sections """
-        email_sections = []
-        for section_key, section_value in sections.items():
-            email_sections.append(Section(section_key, section_value))
-        return email_sections
+    # @staticmethod
+    # def get_sections(sections):
```

Comment:
Since the Sections tag has [deprecated](https://docs.sendgrid.com/for-developers/sending-email/section-tags), we can just remove it.

Reply:
https://github.com/SFDigitalServices/email-microservice-py/pull/5/commits/ac2fd5c36f5c4596779c0a288030eccaa8d42c97